tests/test_nelder_mead.py

```diff
@@ -72,5 +72,8 @@ def test_nelder_mead_max_iter(check_error):
             max_iter=10,
         ),
         func_workchain_name='rosenbrock',
-        exit_status=202
+        exit_status=202,
+        output_port_names=[
```

Comment:
That's great, could you add it to the `test_nelder_mead` also?

Reply:
I'm not entirely sure what you mean. You mean to add the arguments `output_port_names` and `exit_status` to the `check_optimization` fixture? So I can use it in the `test_nelder_mead` test function?
I can do that, just want to understand exactly

Reply:
Yes, exactly -- sorry, I wasn't being very clear.

Reply:
Only the `output_port_names`, the `exit_status` should always be zero in `check_optimization` I think.